srcpkgs/python-jaraco/template

```diff
@@ -0,0 +1,29 @@
+# Template file for 'python-jaraco'
+pkgname=python-jaraco
+version=1.0
+revision=1
+archs=noarch
+create_wrksrc=yes
+pycompile_module="jaraco"
+short_desc="Namespace package declaration for jaraco (Python2)"
+maintainer="bra1nwave <bra1nwave@protonmail.com>"
+license="Public Domain"
+homepage="https://www.voidlinux.org/"
+
+do_install() {
+	for py_sitelib in $py2_sitelib $py3_sitelib; do
+		vmkdir ${py_sitelib}/jaraco
+		cat > ${DESTDIR}/${py_sitelib}/jaraco/__init__.py <<-EOF
```

Comment:
Clever.
As we do not plan to add new py2 modules, this seems useful only for py3.